WpAssets.php

```diff
@@ -325,6 +337,16 @@ public function addInline($script) {
       });
     }
 
+    /**
+     * Add attributes to script tag
+     */
+
+    if ($s['attrs']) {
+      foreach ($s['attrs'] as $attr => $value) {
+          self::addAttr($s['handle'], $attr, $value);
```

Comment:
Looks like there are four spaces to indent this line instead of the required two.